server/Assignments_filler/sheetFiller.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import print_function
+
+import os.path
```

Comment:
```suggestion
import os
```
You use it as `os.path`, just import the module in its entirety.